tests/test_cv.py

```diff
@@ -12,11 +12,24 @@
 # subsample 100 obs
 sc.pp.subsample(adata, n_obs=100, random_state=42)
 
-rf_estimator = create_estimator("random_forest_classifier", Params(random_state=42))
+rf_classifier = create_estimator("random_forest_classifier", Params(random_state=42))
+lr_classifier = create_estimator('logistic_regression_classifier')
```

Comment:
Doesn't this also have a random state? The sklearn implementation has one I thought?

Reply:
True! adding it.